docs/data/joy/customization/dark-mode/DarkModeByDefault.js

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { CssVarsProvider } from '@mui/joy/styles';
+import Sheet from '@mui/joy/Sheet';
+import Chip from '@mui/joy/Chip';
+import Typography from '@mui/joy/Typography';
+
+const useEnhancedEffect =
+  typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;
+
+export default function ButtonThemes() {
+  // the `node` is used for attaching CSS variables to this demo, you might not need it in your app.
```

Comment:
It overflows in the demo:

```suggestion
  // the `node` is used for attaching CSS variables to this demo, you might not
  // need it in your app.
```

<img width="795" alt="Screenshot 2022-10-24 at 12 50 11" src="https://user-images.githubusercontent.com/3165635/197509734-7a12abf4-d070-4f56-9315-8ee444a94d3e.png">